FROST-Server.Core.Model/src/main/java/de/fraunhofer/iosb/ilt/frostserver/query/Metadata.java

```diff
@@ -0,0 +1,21 @@
+package de.fraunhofer.iosb.ilt.frostserver.query;
```

Comment:
Could you add the LGPL license header?
You can leave out the copyright line, set it to yourself if you like, or to Fraunhofer if you don't mind transferring the copyright.
(also on the other new files please)

Reply:
This is fixed.